modules/administration/pages/space-management.adoc

```diff
@@ -0,0 +1,67 @@
+[[space-management]]
+= Managing Disk Space
+
+Running out of disk space can have a severe impact on the {productname} database and file structure which, in most cases, is not recoverable.
+
+{productname} monitors some directories for free disk space.
+You can modify which directories are monitored, and the warnings that are created.
+All settings are configured in the [path]``/etc/rhn/rhn.conf`` configuration file.
+
+
+
+== Monitored Directories
+
+By default, {productname} monitors these directories:
+
+* [path]``/var/lib/pgsql``
+* [path]``/var/spacewalk``
+* [path]``/var/cache /srv``
```

Comment:
Those are two separate directories! /var/spacewalk and /srv

Reply:
oops!